nemo/lightning/io/mixin.py

```diff
@@ -81,15 +81,17 @@ def _partial_representer_with_defaults(dumper, data):
 
 
 def _safe_object_representer(dumper, data):
-    if not inspect.isclass(data):
-        cls = data.__class__
-        call = True
-    else:
+    try:
         cls = data
+        target = f"{inspect.getmodule(cls).__name__}.{cls.__qualname__}"
         call = False
+    except Exception:
```

Comment:
is it possible to specify the exception types here? or is it difficult to enumerate

Reply:
I can change it to `AttributeError`

Reply:
Changed in https://github.com/NVIDIA/NeMo/pull/11106/commits/bd903a0c86b8ba847903f9ba8e1aad37f1f5ea6a